test/data/devicegraphs/multidevice-ext4.xml

```diff
@@ -0,0 +1,1125 @@
+<?xml version="1.0"?>
```

Comment:
I guess this complex xml file comes from a bug report. If that is the case, we usually name the file as "bug_23244.xml". Then we use it for the regression test.

I would rename the file to make clear this is recreating a scenario from a bug. Otherwise, it would look like a overkill xml (too much noise when we only need a ext with two devices).

Reply:
Make sense. Thanks!